### PR TITLE
CU_#860qaa5gw Commingsoon component udpated for library and recipes

### PR DESCRIPTION
**Pull request by Ayomikuolatunji**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,8 +16,8 @@
     "serve": "gatsby serve",
     "clean": "gatsby clean",
     "typecheck": "tsc --noEmit",
-    "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook",
+    "storybook": "npm run download-data && start-storybook -p 6006",
```
Comment (hhimanshu) [inline package.json:19]: Why are we adding back these commands? We decided to remove them

File: static/temp/foods.json [hunk 2]
```diff
@@ -0,0 +1,340 @@
+[
+  {
```
Comment (hhimanshu) [inline static/temp/foods.json:2]: We deleted this file, why are we adding it back?

File: .storybook/main.js [hunk 3]
```diff
@@ -11,26 +12,41 @@ module.exports = {
...
+    config.resolve.modules.push(path.resolve(__dirname, "../data"));
+
     config.resolve.mainFields = ["browser", "module", "main"];
     return config;
   },
   resolve: {
     modules: [path.resolve(__dirname, "src"), "node_modules"],
   },
-  staticDirs: ["../static"],
+  staticDir: "../static",
```
Comment (hhimanshu) [inline .storybook/main.js:49]: I think we do not need this. It breaks the image load

Comment (hhimanshu) [reply .storybook/main.js:49]: See https://www.chromatic.com/test?appId=637ea8c5c336d3dfcb599cdd&id=642a9b2a577b0ea360ebd075

File: src/models/components.d.ts [hunk 4]
```diff
@@ -192,6 +192,7 @@ interface AutoCompleteSearchProps extends onSelectCategoryType {
   categoryOptions: Array<SearchCategoryItemType>;
   recommendations?: Array<recommendationType>;
   setIsMobileSearchOpen: Dispatch<SetStateAction<boolean>>;
+  selectedMobileItem: string;
```
Comment (hhimanshu) [inline src/models/components.d.ts:195]: What is the purpose of this property?

File: static/temp/foods.json (hunk 2 above)
Comment (Ayomikuolatunji) [reply static/temp/foods.json:2]: I just remembered that this component was created from the recnetly view task component and the old code were still there. Fixed anyways.

File: .storybook/main.js (hunk 3 above)
Comment (Ayomikuolatunji) [reply .storybook/main.js:49]: Yes same old code. All is fixed now.

File: src/models/components.d.ts (hunk 4 above)
Comment (Ayomikuolatunji) [reply src/models/components.d.ts:195]: It gives us the selected tab item which can be either Food, Library etc <SearchCategories
                  onSelectCategory={props.onSelectCategory}
                  categoryOptions={props.categoryOptions}
                />

Comment (Ayomikuolatunji) [reply src/models/components.d.ts:195]: Updated to to this selectedSearchCategory

File: package.json (hunk 1 above)
Comment (Ayomikuolatunji) [reply package.json]: fixed

**Review by hhimanshu — APPROVED** [review]

**Pull request merged**